src/FSharp.Data.GraphQL.Server/ReflectionHelper.fs

```diff
@@ -140,6 +140,28 @@ module internal ReflectionHelper =
 
     let isPrameterMandatory = not << isParameterOptional
 
+    let unwrapOptions (ty : Type) =
+        if ty.FullName.StartsWith OptionTypeName || ty.FullName.StartsWith ValueOptionTypeName then
+            ty.GetGenericArguments().[0]
+        else ty
+
+    let isAssignableWithUnwrap (from: Type) (``to``: Type) =
+        let from =
+            if from.FullName.StartsWith OptionTypeName || from.FullName.StartsWith ValueOptionTypeName then
+                from.GetGenericArguments().[0]
+            else from
+        let ``to`` =
+            if ``to``.FullName.StartsWith OptionTypeName || ``to``.FullName.StartsWith ValueOptionTypeName then
+                ``to``.GetGenericArguments().[0]
+            else ``to``
+
+        let result = from.IsAssignableTo ``to``
+        if result then result
+        else
+            if from.FullName.StartsWith OptionTypeName || from.FullName.StartsWith ValueOptionTypeName then
+                from.GetGenericArguments().[0].IsAssignableTo ``to``
+            else result
```

Comment:
When we declare scalar with a validator like this
``` F#
type AddressLine1 = ValidString<Phantom.Address.Line1>

module Address =
    open ValidString
    open Validus.Operators

    let createLine1 : Validator<string, AddressLine1 voption> =
        (allowEmpty ?=> (Check.String.lessThanLen 1000 <+> validateStringCharacters)) *|* ValueOption.map ValidString

let Line1Type : ScalarDefinition<string, AddressLine1 voption> = Define.ValidStringScalar<AddressLine1>("AddressLine1", createLine1, "Address line 1")
```
it appears that the type of scalar becomes `voption<AddressLine1>`
and then if we declare it `Nullable`
``` F#
Define.Input("line1", Nullable Address.Line1Type)
```
it is double wrapped like `option<voption<AddressLine1>>`
The unwrapping itself happes on execution well but in order to perform check here I need to try unwrapping it 2 times